upgrade.py

```diff
@@ -448,6 +448,15 @@ def buildRestoreList(self):
         # NRPE service config
         self.restore_list += ['etc/nagios/nrpe.cfg', {'dir': 'etc/nrpe.d'}]
 
+        # SNMP service config
```

Comment:
Looking at other similar code in the same function many comments refer to the Jira ticket (like `# CA-195388: Preserve /etc/mdadm.conf across upgrades` above). Could you add such reference?

Reply:
Thanks for your remind. I added PR description and jira ticket number in comment.